app/components/navigation/header.tsx

```diff
@@ -53,14 +53,23 @@ export const Header = () => {
                 h={{ base: '26px', md: '40px' }}
               />
               {/* TODO Change Link and Add Logout logic */}
-              <MenuList fontSize={{ base: 'md', md: 'lg' }} bgColor="white">
-                <MenuItem bgColor="white" _hover={{ bgColor: 'gray.100' }}>
+              <MenuList fontSize={{ base: 'md', md: 'lg' }} bgColor={bg}>
+                <MenuItem
+                  bgColor={bg}
+                  _hover={{
+                    bgColor: useColorModeValue('gray.100', 'gray.700')
```

Comment:
Can you delete underline when you hover for better design?

Reply:
I fixed it.
https://github.com/tabi-memo/frontend/pull/13/commits/cb481360a5b2926f47d7d52df51871aea0e3cb19